samples/highcharts/demo/polar-advanced/demo.js

```diff
@@ -0,0 +1,553 @@
+            borderColor: colors[4],
+            backgroundColor: toggleableGradient,
+            innerRadius: '40%'
+        }
+    }, {
+        size: '55%',
+        innerSize: '45%',
+        ...paneOpeningAngles,
+        background: {
+            borderWidth: 0,
+            backgroundColor: toggleableGradient,
+            outerRadius: '75%'
+        }
+
+    // ...And this the one we alter
+    }, {
+        size: '100%',
+        innerSize: '88%',
+        startAngle: 16.5,
+        endAngle: 343.5,
+        background: {
+            borderWidth: 1,
+            borderColor: colors[4],
+            backgroundColor: '#46465C',
+            innerRadius: '55%',
+            outerRadius: '100%'
+        }
+    }],
+    xAxis: [{
+        pane: 0,
+        tickInterval: 1,
+        lineWidth: 0,
+        gridLineWidth: 0,
+        min: 1,
+        max: 26,
+        ...noLabelProp
+    }, {
+        pane: 1,
+        linkedTo: 0,
+        gridLineWidth: 0,
+        lineWidth: 0,
+
+        // We put some plotbands on the chart to represent weekends
+        // when no datapoints occur.
+        plotBands: Array(3).fill(7).map(
+            (weekendOffset, week) => {
+                const
+                    from = weekendOffset * (week + 1),
+                    to = from - 1;
+                return { from, to, color: '#BBBAC5' };
+            }
+        ),
+        ...monthExtremes,
+        ...noLabelProp
+    }, {
+        pane: 2,
+        tickAmount: 4,
+        tickInterval: 0.5,
+        gridLineWidth: 0,
+        lineWidth: 0,
+        ...weekExtremes,
+        ...noLabelProp
+    }],
+    yAxis: [{
+        pane: 0,
+        gridLineWidth: 0.5,
+        gridLineDashStyle: 'longdash',
+        tickInterval: 1,
+        title: null,
+        ...noLabelProp,
+        min: 1,
+        max: 3
+    }, {
+        pane: 1,
+        reversed: true,
+        gridLineWidth: 0,
+        tickInterval: 100,
+        min: 0,
+        max: 400,
+        title: null,
+        ...noLabelProp
+    }, {
+        pane: 2,
+        tickInterval: 0.25,
+        gridLineWidth: 0,
+        gridLineColor: colors[1].brighten(0.05).get('rgba'),
+        min: -3,
+        max: 1,
+        title: null,
+        ...noLabelProp
+    }],
+    legend: {
+        enabled: true,
+        floating: true,
+        layout: 'vertical',
+        verticalAlign: 'center',
+        align: 'center',
+        backgroundColor: '#1f1836',
+        borderColor: 'transparent',
+        borderRadius: 12,
+        itemStyle: {
+            color: '#FFF'
+        },
+        itemHoverStyle: {
+            color: '#BBBAC5'
+        },
+        itemMarginBottom: 1,
+        width: '26%',
+        maxHeight: '14%'
+    },
+    plotOptions: {
+        columnrange: {
+            custom: {
+                textSizeClass: 'small-size'
+            }
+        }
+    },
+    responsive: {
+        rules: [
+            {
+                condition: {
+                    minWidth: 0
```

Comment:
Will do! 

I originally did do this, however (if i recall correctly) it seemed like those styles were not being re-applied when resizing up, then down again 🤔 

I will look into i again though!

Reply:
> it seemed like those styles were not being re-applied when resizing up, then down again

Ok, if that's true it would be a bug. Let me know what you find out.

Reply:
I did a basic test with `legend.borderWidth`, it seems to work correctly: https://jsfiddle.net/highcharts/z4m369kx/